pyproject.toml

```diff
@@ -1,3 +1,84 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mindee"
+authors = [
+  {name = "Mindee", email = "opensource@mindee.com"},
+]
+dynamic = ["version"]
+description = "Mindee API helper library for Python"
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = [
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
```

Comment:
We don't officially support this anymore, I believe

Reply:
https://github.com/mindee/mindee-api-python/blob/main/.github/workflows/unit-test.yml#L18

Reply:
Mmmh, what do you say we get rid of that? :grin:

Reply:
can do for next major version, it's a breaking change.